src/components/pages/farmer-member/edit/index.tsx

```diff
@@ -111,7 +111,7 @@ export default function FarmerEditPageComponent({ edit }) {
       <PageHeading actions={<ActionButtons />}>🧑‍🌾 Edit Farmer</PageHeading>
       <Accordion defaultIndex={[0]} allowMultiple>
         <FarmerEditForm initialData={farmer} handleSubmit={handleSubmit} ref={ref} />
-        <Stack direction={"column"} spacing={2} width={"full"} height={"600px"}>
+        {/* <Stack direction={"column"} spacing={2} width={"full"} height={"600px"}>
```

Comment:
if its not required can we simply remove it?

Reply:
No this is required, However, adding geojson support is breaking the edit marker flow. So in the meantime, it has been disabled.